java/searching/linear_search.java

```diff
@@ -0,0 +1,28 @@
+public class linear_search{
```

Comment:
Class should follow Java class naming convention

Reply:
Now in the recent commit followed all naming conventions...